src/test/resources/test_data/overall_calculation/custom-examples/crs-1070-ual-on-tused.json

```diff
@@ -0,0 +1,355 @@
+{
+  "offender":{
+    "reference":"A8857AD",
```

Comment:
DOB and reference also should probably be changed. You can copy the test one from another file.